webrender/res/ps_border_corner.vs.glsl

```diff
@@ -27,15 +27,19 @@ void set_radii(int style,
         case BORDER_STYLE_RIDGE:
         case BORDER_STYLE_GROOVE:
             vRadii1.xy = radii - adjusted_widths;
-            vRadii1.zw = -widths;
+            // See comment in default branch
+            vRadii1.zw = vec2(-100, -100);
```

Comment:
These will need to be ```vec2(-100.0)``` - some shader compilers require the explicit float to avoid a numeric cast error.